unimport/console.py

```diff
@@ -1,11 +1,63 @@
+import argparse
 import os
-import sys
+import pathlib
 import tokenize
 
+from unimport.config import Config
 from unimport.files import get_files
 from unimport.unused import get_unused
 
 
+class Cli(object):
+    def parse_args(self):
+        parser = argparse.ArgumentParser(
+            description="Detect or remove unused Python imports."
+        )
+        parser.add_argument(
+            "source",
+            default=".",
+            nargs="?",
+            help="include file or folder to find the unused imports",
+        )
+        parser.add_argument(
+            "-c",
+            "--config",
+            help="read configuration from PATH",
+            metavar="PATH",
+            type=argparse.FileType("rb"),
```

Comment:
Like `source` there are no need to use `argparse.FileType`, just `Path` would be enough if I am not missing something.